underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java

```diff
@@ -181,10 +195,9 @@ public static S3AUnderFileSystem createInstance(AlluxioURI uri,
     }
     clientConf.setMaxConnections(numThreads);
 
-    // Set client request timeout for all requests since multipart copy is used, and copy parts can
-    // only be set with the client configuration.
-    clientConf
-        .setRequestTimeout((int) Configuration.getMs(PropertyKey.UNDERFS_S3A_REQUEST_TIMEOUT));
+    boolean streamingUploadEnabled =
+        conf.isSet(PropertyKey.UNDERFS_S3A_STREAMING_UPLOAD_ENABLED)
+        && conf.getBoolean(PropertyKey.UNDERFS_S3A_STREAMING_UPLOAD_ENABLED);
```

Comment:
Do we need to check `isSet` since there is a default value?

Reply:
I am a little confused with the configuration in `S3AUnderFileSystem`. Some configuration comes from the `UnderFileSystemConfiguration` and some come from the Global `Configuration`.

For here, `isSet` is not needed, I will remove it, thanks